src/Components/HelloWorldPage/index.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from "react";
+import * as S from "./style";
+import * as I from "../../Assets/SVG";
+// import axios from "axios";
+
+// const joinData: Array<String> = [];
+// const loginData: Array<String> = [];
+
+const SignUp: React.FC = () => {
+  const value = [
+    { name: "아이디", placeholder: "영문, 숫자를 포함한 5자 이상" },
+    {
+      name: "비밀번호",
+      placeholder: "영문, 숫자를 포함한 4~20자 이내",
+      type: "password",
+      maxLength: 20,
+      minLength: 4,
+    },
+    {
+      name: "비밀번호 확인",
+      placeholder: "",
+      type: "password",
+      maxLength: 20,
+      minLength: 4,
+    },
+    {
+      name: "닉네임",
+      placeholder: "특수문자를 제외한 10자 이내",
+      maxLength: 10,
+    },
+  ];
+
+  return (
+    <>
+      {value.map((value, index) => {
+        return (
+          <div key={index}>
+            <S.Name>{value.name}</S.Name>
+            <S.Input
+              placeholder={value.placeholder}
+              type={value.type}
+              maxLength={value.maxLength}
+              minLength={value.minLength}
+              // onChange={(e) => (joinData[index] = e.target.value)}
+            ></S.Input>
+          </div>
+        );
+      })}
```

Comment:
```suggestion
      {value.map((value, index) => (
          <div key={index}>
            <S.Name>{value.name}</S.Name>
            <S.Input
              placeholder={value.placeholder}
              type={value.type}
              maxLength={value.maxLength}
              minLength={value.minLength}
              // onChange={(e) => (joinData[index] = e.target.value)}
            ></S.Input>
          </div>
        );
      )}
```

map함수를 쓸때도 `() => ()` 로 사용하는게 더 보기 좋겠네요